udark/udapp/src/main/java/io/underdark/app/model/Node.java

```diff
@@ -151,12 +156,12 @@ public void transportLinkConnected(Transport transport, Link link)
 		routingTable.put(link.getNodeId(),new RoutingInfo(link.getNodeId(),1));
 		activity.refreshPeers();
 		activity.refreshButtons();
-		String rt = encodeRoutingTable();
+		//String rt = encodeRoutingTable();
 		//now send encoded routing table to each adjacent node
-		for (Link l:
+		/*for (Link l:
```

Comment:
Same question as previous comment. We should be sending this routing info to all links.